packages/sdks/python/tests/test_client.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from generated.client import Metriport
```

Comment:
@dsinghvi what is generated? is this a fern specific thing?

Reply:
Yup checkout `generators.yml` the sdks are locally generated to the `generated` folder -- the reason for this is so that you can then reference the client from the generated folder

Reply:
It's the easiest way to have non generated and generated code live together

Reply:
ok so when I followed the readme I got erorrs the their is no module named generated. Are there certain fern commands I have to run before running `poetry run pytest tests` to get this to work?

Reply:
yes you have to run `fern generate --group test` and that will dump the fern sdks in the generated folders